src/app/showcase/showcase.routes.ts

```diff
@@ -42,6 +43,10 @@ export const routes: Routes = [
         path: 'list',
         component: ListShowcaseComponent,
       },
+      {
```

Comment:
Move to `list/load-on-demand`
```suggestion
      {
        path: 'list',
        component: ListShowcaseComponent,
        children: [
          {
            path: 'load-on-demand',
            component: ListLoadOnDemandShowcaseComponent,
          },
        ],
      },
```

Reply:
As we talked about, the menu styling does not support subroutes as we speak, and i don't want to makes changes for that in this PR.